### PR TITLE
chore(dev): radix tooltip

### PR DESCRIPTION
**Pull request by jo-fang**

### CODE REVIEW

File: weave-js/src/components/RadixTooltip/TooltipAnatomy.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
+import * as RadixTooltip from '@radix-ui/react-tooltip';
```
Comment (jamie-rasmussen) [inline weave-js/src/components/RadixTooltip/TooltipAnatomy.tsx:1]: The filename here seems a little odd, could it be RadixTooltip.tsx?

**Comment by jo-fang** [comment]
Copied file from [OrgDashboard/UserActivity/components/Tooltip](https://github.com/wandb/core/blob/371b1c8d67a6d5c0c4ba2ae62360e2899edaf7a2/frontends/app/src/pages/OrgDashboard/UserActivityDashboard/components/Tooltip.tsx) file. I'm finally doing what we should've done months ago

File: weave-js/src/components/RadixTooltip/TooltipAnatomy.tsx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+import {twMerge} from 'tailwind-merge';
+
+import {Tailwind} from '../Tailwind';
+
+/**
+ * {@link https://www.radix-ui.com/docs/primitives/components/tooltip#trigger}
+ */
+export const Provider = (props: RadixTooltip.TooltipProviderProps) => (
+  <RadixTooltip.Provider
+    delayDuration={250}
```
[the remark this answers is not in the document]
Comment (jo-fang) [reply weave-js/src/components/RadixTooltip/TooltipAnatomy.tsx:13]: Cecile asked for 400 instead haha

File: weave-js/src/components/RadixTooltip/TooltipAnatomy.tsx (hunk 1 above)
Comment (jo-fang) [reply weave-js/src/components/RadixTooltip/TooltipAnatomy.tsx]: i moved everything into index.tsx so i don't have to worry about filename :D

File: weave-js/src/components/RadixTooltip/TooltipAnatomy.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (jamie-rasmussen) [reply weave-js/src/components/RadixTooltip/TooltipAnatomy.tsx]: lol, @Cecile0112358 is definitely the person who told me 500 before ;-)
https://weightsandbiases.slack.com/archives/C03BSTEBD7F/p1697224652032549?thread_ts=1697219819.003369&cid=C03BSTEBD7F

File: weave-js/src/components/RadixTooltip/TooltipAnatomy.tsx (hunk 1 above)
Comment (jamie-rasmussen) [reply weave-js/src/components/RadixTooltip/TooltipAnatomy.tsx]: Do we have a frontend style guideline on when to use a single file vs. a directory with a single index.tsx file?

**Review by jamie-rasmussen — APPROVED** [review]

File: weave-js/src/components/RadixTooltip/TooltipAnatomy.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (jo-fang) [reply weave-js/src/components/RadixTooltip/TooltipAnatomy.tsx]: @jamie-rasmussen  i changed it back to 500 per Cecile in dms lmao

File: weave-js/src/components/RadixTooltip/TooltipAnatomy.tsx (hunk 1 above)
Comment (jo-fang) [reply weave-js/src/components/RadixTooltip/TooltipAnatomy.tsx]: @jamie-rasmussen not that i'm aware of. do you have a suggested standard we should start doing? I've only run into this for common components and can't decide what the best method is

Comment (jamie-rasmussen) [reply weave-js/src/components/RadixTooltip/TooltipAnatomy.tsx]: My inclination would be to just have a single RadixTooltip.tsx file but not sure what the argument for the other possibility is.

Comment (jo-fang) [reply weave-js/src/components/RadixTooltip/TooltipAnatomy.tsx]: @jamie-rasmussen I was going to move the existing [`Tooltip.tsx`](https://github.com/wandb/weave/blob/bf140c4209ce39733b2dbddfc6f5d0e7fb1044c4/weave-js/src/components/Tooltip.tsx#L12) into this folder to group them all together.

I was thinking
- [ ] Replace current semantic Tooltip.tsx with Radix parts
- [ ] Rename folder from `RadixTooltip` to `Tooltip` and move the ergonomic `Tooltip.tsx` file into folder too

am i overthinking this? I could totally shove the ergonomic tooltip with the different radix parts into same file

Comment (jamie-rasmussen) [reply weave-js/src/components/RadixTooltip/TooltipAnatomy.tsx]: I see, that plan sounds reasonable to me.

**Pull request merged**